EventListener/RouteAnnotationEventListener.php

```diff
@@ -214,6 +274,53 @@ protected function getUrlConcrete($name, $options)
         }
     }
 
+    /**
+     * @param string $name    Route name
+     * @param array  $options Node options
+     *
+     * @throws \InvalidArgumentException
+     * @return UrlConcrete
+     */
+    protected function getMultilangUrl($name, $options)
+    {
+        try {
+            $params = [];
+
+            if ($options['default_locale']) {
+                $params['_locale'] = $options['default_locale'];
+            }
+
+            $url = new UrlConcrete(
+                $this->getRouteUri($name, $params),
+                $options['lastmod'],
+                $options['changefreq'],
+                $options['priority']
+            );
+
+            if ($options['locales'] && is_array($options['locales'])) {
+                $url = new GoogleMultilangUrlDecorator($url);
+
+                foreach ($options['locales'] as $locale) {
+                    $params['_locale'] = $locale;
+
+                    $url->addLink($this->getRouteUri($name, $params), $locale);
+                }
+            }
+
+            return $url;
+        } catch (\Exception $e) {
```

Comment:
Variables should never be that short, please consider using `$exception` instead